frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -29,19 +29,16 @@ const MonthlyWorkingHours = () => {
           transformedData[item.year].total += item.total_hours;
         }
         setData(Object.values(transformedData));
-      })
-      .catch((error) => {
-        console.error("Error!", error);
       });
   }, []);
 
   const columns = [];
   columns.push({ title: "Vuosi", dataIndex: "year", key: "year" });
-  for (let i = 0; i < 12; i++) {
+  for (let i = 1; i < 13; i++) {
     columns.push({
-      title: `${i + 1}`,
-      dataIndex: `${i + 1 < 10 ? `0${i + 1}` : i + 1}`,
-      key: `${i + 1}`,
+      title: i,
+      dataIndex: i < 10 ? `0${i}` : i,
```

Comment:
Would be best if we ensure i is of the same data type in both cases. Could you convert it to a string in the case when i >= 10